File: leetcode/C++/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        return stable_sort(nums.begin(),nums.end());
    }
};


/*
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> v(3,0);
        for(int &n : nums) ++v[n];
        for(int i=0,k=0; i<v.size(); ++i){
            for(int j=0; j<v[i]; ++j){
                nums[k++]=i;
            }
        }
    }
};
*/

File: leetcode/Python/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        nums.sort()
        return nums

'''
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        v,idx = [0]*3, 0
        for n in nums:
            v[n]+=1
        for i in range(len(v)):
            for j in range(v[i]):
                nums[idx], idx = i, idx+1
        
'''